test/helpers/data_sets_helper_test.rb

```diff
@@ -0,0 +1,171 @@
+require 'test_helper'
+
+class DataSetsHelperTest < ActionView::TestCase
+  include DataSetsHelper
+
+  def assert_similar_arrays(a, b)
```

Comment:
The name `assert_similar_arrays` is kind of misleading.  Really this only checks similar lengths.  A function that asserts similar arrays sounds like it also checks the contents.